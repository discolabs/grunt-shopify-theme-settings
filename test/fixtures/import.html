<fieldset>
  <legend>Appearance &amp; Fonts</legend>

  <h3>Background</h3>
  <table>
    <tr>
      <td><label for="body_background_image">Background Image</label></td>
      <td>
        <select name="body_background_image" id="body_background_image">
          <option value="none">No Image</option>
          <option value="custom_background_image">Use Custom Image</option>
        </select>
      </td>
    </tr>

    <tr>
      <td></td>
      <td>
        <input type="file" id="custom_background_image" name="custom_background_image"  />
      </td>
    </tr>

    <tr>
      <td><label for="body_background_x">Background Horizontal Position</label></td>
      <td>
        <select id="body_background_x" name="body_background_x">
          <option value="0%">Left</option>
          <option value="50%">Center</option>
          <option value="100%">Right</option>
        </select>
      </td>
    </tr>

    <tr>
      <td><label for="body_background_y">Background Vertical Position</label></td>
      <td>
        <select id="body_background_y" name="body_background_y">
          <option value="0%">Top</option>
          <option value="50%">Center</option>
          <option value="100%">Bottom</option>
        </select>
      </td>
    </tr>

    <tr>
      <td><label for="body_background_repeat">Background Tiling</label></td>
      <td>
        <select name="body_background_repeat">
          <option value="no-repeat">Don't Tile</option>
          <option value="repeat">Tile</option>
          <option value="repeat-x">Tile Horizontally</option>
          <option value="repeat-y">Tile Vertically</option>
        </select>
      </td>
    </tr>

    <tr>
      <td><label for="body_background_color">Background Color</label></td>
      <td>
        <input id="body_background_color" name="body_background_color" class="color" value="#FFFFFF" />
      </td>
    </tr>
  </table>