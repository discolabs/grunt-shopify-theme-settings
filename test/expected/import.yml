"Appearance & Fonts":
  Background:
    Background Image:
      name: body_background_image
      type: select
      options:
        none: No Image
        custom_background_image: Use Custom Image
    "Untitled Field #2":
      name: custom_background_image
      type: file
    Background Horizontal Position:
      name: body_background_x
      type: select
      options:
        "0%": Left
        "50%": Center
        "100%": Right
    Background Vertical Position:
      name: body_background_y
      type: select
      options:
        "0%": Top
        "50%": Center
        "100%": Bottom
    Background Tiling:
      name: body_background_repeat
      type: select
      options:
        no-repeat: "Don't Tile"
        repeat: Tile
        repeat-x: Tile Horizontally
        repeat-y: Tile Vertically
    Background Color:
      name: body_background_color
      type: color
    Background Name:
      name: body_background_name
      type: text-single
      default: Default Name
